client/scripts/views/pages/profile/profile_header.ts

```diff
@@ -31,7 +32,17 @@ const editIdentityAction = (account, currentIdentity: SubstrateIdentity, vnode)
     // wait for info to load before making it clickable
     disabled: vnode.state.chainLoading,
     onclick: async () => {
-      if (!app.chain?.loaded) {
+      const chain = chainObj.name.toLowerCase();
```

Comment:
Are you confident "name" is the right property to compare in this case? `app.activeId()` returns `app.chain.id`...

Reply:
Yeah you're totally right, it should be id.